docs/scripts/assets.ts

```diff
@@ -0,0 +1,104 @@
+import { promises as fs } from 'fs'
+import fg from 'fast-glob'
+
+const font = 'https://fonts.googleapis.com/css2?family=Readex+Pro:wght@200;400;600&display=swap'
+
+const googleapis = 'https://fonts.googleapis.com'
+const gstatic = 'https://fonts.gstatic.com'
+const jsdelivr = 'https://cdn.jsdelivr.net/'
+const antfu = 'https://antfu.me'
+const patak = 'https://patak.dev'
+const github = 'https://github.com'
+const avatars = 'https://avatars.githubusercontent.com'
+
+const preconnect = `
+    <link rel="dns-prefetch" href="${googleapis}">
+    <link rel="dns-prefetch" href="${gstatic}">
+    <link rel="preconnect" crossorigin="anonymous" href="${googleapis}">
+    <link rel="preconnect" crossorigin="anonymous" href="${gstatic}">
+`
+const preconnectHome = `
+    <link rel="dns-prefetch" href="${googleapis}">
+    <link rel="dns-prefetch" href="${gstatic}">
+    <link rel="dns-prefetch" href="${antfu}">
+    <link rel="dns-prefetch" href="${patak}">
+    <link rel="dns-prefetch" href="${jsdelivr}">
+    <link rel="dns-prefetch" href="${github}">
+    <link rel="dns-prefetch" href="${avatars}">
+    <link rel="preconnect" crossorigin="anonymous" href="${googleapis}">
+    <link rel="preconnect" crossorigin="anonymous" href="${gstatic}">
+    <link rel="preconnect" crossorigin="anonymous" href="${antfu}">
+    <link rel="preconnect" crossorigin="anonymous" href="${patak}">
+    <link rel="preconnect" crossorigin="anonymous" href="${jsdelivr}">
+    <link rel="preconnect" crossorigin="anonymous" href="${github}">
+    <link rel="preconnect" crossorigin="anonymous" href="${avatars}">
+`
```

Comment:
Can we automate this? I am not so confident of duplicating (It would make future changes much harder)

Reply:
We can include some module with the sita data, including all stuff there: avatars, links, contributors, fonts...

I'll be back next Tuesday, we can merge and then change it, but proceed as you wish